src/Components/Search.js

```diff
@@ -0,0 +1,5 @@
+import React from "react";
+
+export const Search = ({ onChange, value }) => {
+  return <input value={value} onChange={e => onChange(e)} />;
```

Comment:
Be careful here. When binding an input like this, anyone can pass any function here. Like an API call 😉. What happens is that when a user types, they can keep typing, and each type will fire an independent API call. This means for `hello` we have 5 API requests and they all return at different times because APIs take their own time to return.

One way to fix this problem is to use a function that keeps track, or holds on to the call for a certain amount of time, until either, the timer runs out, or the function is called again. This is traditionally called a `debounce` function. Its pretty normal to bind one to an input.